DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/IdentityExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Security.Claims;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi
+{
+    public static class IdentityExtensions
+    {
+        public static string GetUserId()
+        {
+            var userId = ClaimsPrincipal.Current.Identity.GetUserId();
```

Comment:
Please contact to https://github.com/alexrrrr how to correct write identity extension.